.github/workflows/run-tests.yml

```diff
@@ -145,21 +176,46 @@ jobs:
         run: |
           docker compose -f ${{ env.DOCKER_COMPOSE_FILE }} down -v
 
+
+  set-up-functional-test:
+    if: always()
```

Comment:
You don't need this here

Reply:
previously it was skipped until I added that. Not sure whether it's needed anymore because I've changed a lot of things, but that's why it's there